### PR TITLE
update windows compilers

### PR DESCRIPTION
**Pull request by ahartikainen**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

This should now add m2w64-toolchain for build and run.

### CODE REVIEW

**Comment by ahartikainen** [comment]
@conda-forge-admin, please rerender

**Comment by ahartikainen** [comment]
Hi, anyone knows why it tried to run gtests?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ requirements:
     - {{ compiler('cxx') }}  # [not win]
     - {{ compiler('m2w64_cxx') }}  # [win]
     - python
-    - {{ pin_compatible('numpy') }}
```
Comment (beckermr) [inline recipe/meta.yaml]: The pinning statement should stay. Why did this get removed?

Comment (beckermr) [reply recipe/meta.yaml]: If you need a min version of numpy, you should add a constraint in the host section like this `numpy 1.15.*` or whatever. See the docs here https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Comment (ahartikainen) [reply recipe/meta.yaml]: it build against 1.11. Is that normal with conda-forge?

Comment (beckermr) [reply recipe/meta.yaml]: Yes. The build is against an older version so that it will run against any newer version past that.

Comment (ahartikainen) [reply recipe/meta.yaml]: Ok, I can rewind it back

Comment (beckermr) [reply recipe/meta.yaml]: You should always build against the oldest version possible down to 1.11.*. However, if the package can only run on say 1.14.* and later, then by following the instructions in the link above, the `pin_compatible` statement will automatically put in the right dependency range in the run section.

**Comment by ahartikainen** [comment]
Ok, it works without gtests. Should we run them anyway? Have gtest folder just changed location?

**Comment by ahartikainen** [comment]
I guess this is ready

**Comment by beckermr** [comment]
LGTM! I double checked that the CI actually compiled and ran the models in each case. Looks like it all worked.

**Pull request merged**